tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from openmc import _utils
+import pytest
+
+# from tests import cdtemp
+
+
+# @pytest.fixture(scope="module")
+# def download_photos():
+@pytest.fixture()
+def download_photos(run_in_tmpdir):
+    """use _utils download() function to download the same picture four times,
+       twice to get unique names, & a third time to use the already downloaded
+       block of code, and a fourth time to use the checksum block"""
+    # _utils.download("https://media.gettyimages.com/photos/" +
+    #                 "flamingo-picture-id957054810?s=2048x2048")
+    # with cdtemp():
+    _utils.download("https://i.ibb.co/HhKFc8x/small.jpg")
+    _utils.download("https://tinyurl.com/y4t38ugb")
+    _utils.download("https://tinyurl.com/y4t38ugb", as_browser=True)
+
+
+# @pytest.fixture(scope="module")
+# def get_checksum_error():
+@pytest.fixture()
+def get_checksum_error(run_in_tmpdir):
+    """use download() in such a way that will test the checksum error line"""
+    phrase = "MD5 checksum for y4t38ugb does not match. If this is your "
+    "first time receiving this message, please re-run the script. "
+    "Otherwise, please contact OpenMC developers by "
+    "emailing openmc-users@googlegroups.com."
+    # with cdtemp():
+    with pytest.raises(OSError, match=phrase):
+        _utils.download("https://tinyurl.com/y4t38ugb", as_browser=True,
+                        checksum="not none")
+
+
+def test_photos(download_photos):
+    assert os.path.getsize("small.jpg") == \
+           os.path.getsize("y4t38ugb")
```

Comment:
`filecmp.cmp("small.jpg", "yt38ub")` would be a little more robust (actually comparing content, not just size)